sdk/keyvault/azsecrets/README.md

```diff
@@ -0,0 +1,272 @@
+# Azure Key Vault Secrets client library for Go
+Azure Key Vault helps securely store and control access to tokens, passwords, certificates, API keys, and other secrets.
+
+[Source code][secret_client_src] | [API reference documentation][reference_docs] | [Product documentation][keyvault_docs]
+
+## Getting started
+
+### Install packages
+Install [azure-keyvault-secrets][package_link] and [azure-identity][azure_identity_goget]:
+```
+go get -u github.com/Azure/azure-sdk-for-go/sdk/keyvault/azsecrets
+```
+[azure-identity][azure_identity] is used for Azure Active Directory authentication as demonstrated below.
+```
+go get -u github.com/Azure/azure-sdk-for-go/sdk/azidentity
+```
+
+
+### Prerequisites
+* An [Azure subscription][azure_sub]
+* Go version 1.16 or later
+* A Key Vault. If you need to create one, you can use the [Azure Cloud Shell][azure_cloud_shell] to create one with these commands (replace `"my-resource-group"` and `"my-key-vault"` with your own, unique names):
+
+  (Optional) if you want a new resource group to hold the Key Vault:
+  ```sh
+  az group create --name my-resource-group --location westus2
+  ```
+
+  Create the Key Vault:
+  ```Bash
+  az keyvault create --resource-group my-resource-group --name my-key-vault
+  ```
+
+  Output:
+  ```json
+  {
+      "id": "...",
+      "location": "westus2",
+      "name": "my-key-vault",
+      "properties": {
+          "accessPolicies": [...],
+          "createMode": null,
+          "enablePurgeProtection": null,
+          "enableSoftDelete": null,
+          "enabledForDeployment": false,
+          "enabledForDiskEncryption": null,
+          "enabledForTemplateDeployment": null,
+          "networkAcls": null,
+          "provisioningState": "Succeeded",
+          "sku": { "name": "standard" },
+          "tenantId": "...",
+          "vaultUri": "https://my-key-vault.vault.azure.net/"
+      },
+      "resourceGroup": "my-resource-group",
+      "type": "Microsoft.KeyVault/vaults"
+  }
+  ```
+
+  > The `"vaultUri"` property is the `vault_url` used by [azsecrets.NewClient][secret_client_docs]
```

Comment:
I'm guessing the field is more like VaultURL, right? vault_url would be a bit more pythonic than I would expect.